core/app/models/spree/store_credit.rb

```diff
@@ -116,14 +127,21 @@ def void(authorization_code, options = {})
 
         amount_authorized: amount_authorized - auth_event.amount
       })
+
+      Spree::StoreCreditLedgerEntry.credit(
+        self,
+        auth_event.amount,
+        options[:action_originator],
+        Spree::StoreCreditLedgerEntry::PENDING
+      )
       true
     else
       errors.add(:base, Spree.t('store_credit.unable_to_void', auth_code: authorization_code))
       false
     end
   end
 
-  def credit(amount, authorization_code, order_currency, options = {})
+  def credit(amount, authorization_code = nil, order_currency = "USD", options = {})
```

Comment:
I'm uneasy about the default currency being `USD` here. If this isn't passed in in a context where the currency is anything else, this should actually throw an error IMO.